arch/arm/soc/nxp_imx/rt/Kconfig.defconfig.mimxrt1052

```diff
@@ -19,22 +19,22 @@ config CLOCK_CONTROL_MCUX_CCM
 endif # CLOCK_CONTROL
 
 config INIT_ARM_PLL
-    def_bool y
+	def_bool y
```

Comment:
Move white spaces into original patch that added this stuff

Reply:
sure.